java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/CheckVerifier.java

```
@@ -239,7 +239,9 @@ private static RemediationFunction remediationFunction(AnalyzerMessage issue) {
       }
     } catch (IOException | JsonParseException e) {
```

Exception is ignored intentionally.